### PR TITLE
WIP: use eslint-plugin-patternfly-react

### PR DESCRIPTION
**Pull request by sharvit**
I believe patternfly-react created a great eslint configuration and we should use and embrace it across plugins.

Most of the errors (if not all) is about `propTypes` and `defaultProps`, we just don't use them and we really need to.

Any thoughts about how to do this migration effectiveness?
@amirfefer @ohadlevy

### CODE REVIEW

File: webpack/assets/javascripts/compute_resource/ovirt.js [hunk 1]
```diff
@@ -18,17 +18,21 @@ export function templateSelected(item) {
...
+          .remove();
+        $.each(result.interfaces, function() {
           addNetworkInterface(this);
         });
         $('#storage_volumes .children_fields >.fields').remove();
-        $.each(result.volumes, function () {
+        $.each(result.volumes, function() {
           // Change variable name because 'interface' is a reserved keyword.
-          this.disk_interface = this['interface'];
+          this.disk_interface = this.interface;
```
Comment (ohadlevy) [inline webpack/assets/javascripts/compute_resource/ovirt.js]: imho interface is a reserved word.

**Comment by ohadlevy** [comment]
:+1: overall.

does this work with hound correctly? (notice the error message from hound above)

**Review by amirfefer — COMMENTED** [review]
:+1:  I like the combination with `prettier`
`npm run test` results with tones of errors, 
maybe we can ignore some files, and apply these rules on new code ?

**Comment by sharvit** [comment]
@ohadlevy It is not working with houndci correctly, see https://github.com/houndci/hound/issues/1540

**Comment by sharvit** [comment]
Any Idea how we can ignore those linting error on "unchanged files"?

**Comment by sharvit** [comment]
@houndci-bot Is working and running eslint-patternfly-react! Thanks @gylaz :+1:

**Comment by sharvit** [comment]
All the @houndci-bot comments I marked with :-1: are weird and not happening locally.

**Comment by ohadlevy** [comment]
@sharvit what is the status of the PR?

File: webpack/assets/javascripts/compute_resource/ovirt.js [hunk 2]
```diff
@@ -99,8 +109,13 @@ function addNetworkInterface({ name, network }) {
 
 // fill in the template volumes.
 function addVolume({
-// eslint-disable-next-line camelcase
-  size_gb, storage_domain, bootable, id, disk_interface, wipe_after_delete,
+  // eslint-disable-next-line camelcase
```
Comment (ohadlevy) [inline webpack/assets/javascripts/compute_resource/ovirt.js]: i think you need to change it to a multi line ignore

**Comment by sharvit** [comment]
Discussion to replace hound:
https://community.theforeman.org/t/replacing-hound-ci-for-the-js-stack/11116

**Comment by ohadlevy** [comment]
Why? It seems the main blocker (hound) was resolved?

**Comment by sharvit** [comment]
@ohadlevy It is to messy here. I will open a new PR soon.
I actually managed to fix manually all the linting errors already in the last commit.

**Pull request closed without merging**